velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -309,6 +318,9 @@ std::shared_ptr<const ParquetTypeWithId> ReaderBase::getParquetColumnInfo(
           VELOX_CHECK_EQ(children.size(), 1);
           const auto& child = children[0];
           auto grandChildren = child->getChildren();
+          // This level will not be repeated since parquet will have a child layer with the repeated info which we are ignoring here
```

Comment:
could you elaborate more? it's a bit confusing that quote "This level will not be repeated" and next line `isRepeated = true`